app.py

```diff
@@ -8,6 +8,7 @@
 mongo = PyMongo(app)
 
 from routes.user import *
+from routes.post import *
```

Comment:
this imports the routes from routes/post.py right?

Reply:
yes